cms/core/tests/test_permissions.py

```diff
@@ -0,0 +1,102 @@
+from django.conf import settings
+from django.contrib.auth.models import Group
+from django.test import TestCase
+
+from cms.users.models import User
+
+WAGTAIL_PERMISSION_TYPES = ["add", "change", "delete"]
+
+WAGTAIL_PAGE_PERMISSION_TYPES = ["add", "change", "bulk_delete", "lock", "publish", "unlock"]
+
+
+class PermissionsTestCase(TestCase):
+    @classmethod
+    def create_user(cls, first_name, last_name, group_name):
+        # Create user
+        cls.user = User(first_name=first_name, last_name=last_name)
+        cls.user.save()
```

Comment:
q: why not use `UserFactory`?
```python
cls.user = UserFactory(first_name=first_name, last_name=last_name)
```

Reply:
Omission from when I was first writing the tests, thanks for spotting this